modules/juce_audio_devices/native/juce_ios_Audio.cpp

```diff
@@ -982,6 +1041,12 @@ struct iOSAudioIODevice::Pimpl      : public AudioPlayHead,
                 
         setAnalogInputGain(0.5f);
         
+        if (isUsingBuiltInSpeaker() && !AudioIODeviceType::useDeviceVoiceProcessing) {
+            setAudioPreprocessingEnabled(true);

```

Comment:
Isn't it true by default?

Reply:
I'm getting a different result if I don't call this function. I think `AVAudioSessionModeDefault` is the default value